main.cpp

```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
+//  Noun:       screwdriver
+//  action 1:   the screwdriver screws 
+screwdriver.screw();
+//  action 2:   the screwdriver unscrews
+screwdriver.unscrew();
+//  action 3:   the screwdriver strips
+screwdriver.strip();
 
-//  action 2:
-
-//  action 3:
- 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun:       torch
+//  action 1:   the torch burns
+torch.burn();
+//  action 2:   the torch lights
+torch.light();
+//  action 3:   the torch shows the direction of wind
+torch.showDirectionOfWind();
 
-//  action 3:
- 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       dog
+//  action 1:   the dog eats
+dog.eat();
+//  action 2:   the dog fetches
+dog.fetch();
+//  action 3:   the dog finds you
+dog.findsYou();
```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**replace** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.